tests/unit/utility.test.ts

```diff
@@ -0,0 +1,145 @@
+import {
+  compareUint8Arrays,
+  createRangeFromLength,
+  customShiftValue,
+  delay,
+  isDefined,
+  isUndefined,
+  reverseBytes,
+  shiftValue,
+  truncateAddress,
+  unshiftValue,
+} from '../../src/utilities';
+
+describe('Utility Functions', () => {
+  describe('shiftValue', () => {
+    it('correctly shifts the value by a power of 10^8', () => {
+      const value = 1;
+      const shiftedValue = shiftValue(value);
+      expect(shiftedValue).toBe(100000000);
+    });
+  });
+
+  describe('unshiftValue', () => {
+    it('correctly unshifts the value by a power of 10^8', () => {
+      const value = 100000000;
+      const unshiftedValue = unshiftValue(value);
+      expect(unshiftedValue).toBe(1);
+    });
+  });
+
+  describe('customShiftValue', () => {
+    it('correctly shifts the value based on the provided parameters', () => {
+      const value = 1;
+      const shift = 8;
```

Comment:
Is 8 the default number when the shift parameter is not included? if so, use a different one to make sure it's really using the input.

Reply:
There isn't any default value, it is not an optional field, but I can change the shift value so it would be different than ``shiftValue()`` and ``unshiftValue()``.